web_console/odc-about-web-terminal.adoc

```diff
@@ -15,6 +15,14 @@ Cluster administrators can access the web terminal in {product-title} 4.7 and la
 
 This terminal instance is preinstalled with common CLI tools for interacting with the cluster, such as `oc`, `kubectl`,`odo`, `kn`, `tkn`, `helm`, `kubens`, and `kubectx`. It also has the context of the project you are working on and automatically logs you in using your credentials.
 
+The following new features have been introduced in the web terminal:
+
+* Adding multiple tabs - Support is provided for adding multiple tabs within the Web Terminal in the console. By default, a single tab appears when you open the Web Terminal. You can open multiple tabs by clicking the *+* button on the terminal.
+
+* Enable history - The new tab persists history within the terminal. You can use the terminal, leave it and return after some time but the terminal retains the history.
```

Comment:
Terminal retains bash history i.e. list of commands that were executed, not output of executed commands.

Reply:
After discussing further with @sahil143 in GChat, i have slightly modified the content of the Enable history feature with respect to his feedback. Changes have been made and hence proceeding for QA review.